rcl/test/CMakeLists.txt

```diff
@@ -221,6 +221,15 @@ function(test_target_function)
     AMENT_DEPENDENCIES ${rmw_implementation} "osrf_testing_tools_cpp"
   )
 
+  rcl_add_custom_gtest(test_wait_set_impl${target_suffix}
+    SRCS rcl/test_wait_set_impl.cpp
+    ENV ${rmw_implementation_env_var}
+    APPEND_LIBRARY_DIRS ${extra_lib_dirs}
+    LIBRARIES ${PROJECT_NAME}
+    AMENT_DEPENDENCIES ${rmw_implementation} "osrf_testing_tools_cpp"
+  )
+
+
```

Comment:
nitpick: minimize vertical whitespace (https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace)

Reply:
This was marked as resolved, but was not actually.